src/rgw/rgw_common.h

```diff
@@ -726,6 +726,7 @@ struct rgw_bucket {
   std::string oid; /*
                     * runtime in-memory only info. If not empty, points to the bucket instance object
                     */
+  std::string name_space;
```

Comment:
since namespace is a C++ keyword, it does not allow me to use that as a variable name. Should I change the api name instead?